tests/models/CMakeLists.txt

```diff
@@ -9,6 +9,16 @@ ENDIF()
   LIST(APPEND SHO_SCALARS "potential" "0.225 0.0028")
   LIST(APPEND SHO_SCALARS "latdev"    "1.0   0.013")
 
+# The errors on the total energy are numerical noise, not statistical errors
+IF(QMC_MIXED_PRECISION)
+  LIST(APPEND GEP_SCALARS "totenergy" "0.45  0.00000004")
+ELSE()
+  LIST(APPEND GEP_SCALARS "totenergy" "0.45  4.0e-14")
+ENDIF()
```

Comment:
@kryczko 
The `IF(QMC_MIXED_PRECISION)` seems unnecessary. I copy pasted from SHO test.
Just generate a reasonable error bar for mixed precision build and keep one line for both cases.